packages/twenty-front/src/modules/ui/input/button/components/Button.tsx

```diff
@@ -283,11 +283,9 @@ const StyledButton = styled('button', {
                       : theme.background.transparent.medium
                   }`
                 : 'none'};
-              opacity: ${disabled ? 0.24 : 1};
```

Comment:
Why removing this part @BKM14? Opacity is set to 24% in Figma

![CleanShot 2024-10-25 at 10 16 25](https://github.com/user-attachments/assets/4f951775-ae37-4625-b03c-110ac0f9cb60)

Reply:
Hey, sorry about that. I'm new to this repository and didn't know figma files were available too. I wasn't able to see the borders too well with the button disabled, hence I increased the opacity. I'll change it back to 0.24

Reply:
Now worries, thanks! Did you find access to the Figma file? https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=34844-84769&t=JmWAfjD0ufScVlpq-1

Reply:
Yes, I have reverted the opacity to 0.24 and committed the changes.
Please review it.
Thank you.